client/src/components/ProfileDropdown/ProfileDropdown.js

```diff
@@ -41,7 +40,11 @@ const ProfileDropdown = ({ user }) => {
       {isLoggedOut && <Redirect to={Routes.LOGIN_PAGE} />}
 
       {/** Rendered JSX */}
-      <img alt="Blank Profile" src={blankProfilePicture} className="avatar" />
+      <p>
+        Hello, {user.firstName}!
+        {/* <img alt="Blank Profile" src={blankProfilePicture} className="avatar" /> */}
+        <div className="avatar">{user.firstName?.[0] ?? '!'}</div>
```

Comment:
Why '!' as the default value? Maybe something more explicit would be better here?